### PR TITLE
feat: add support for an empty destination to swap

### LINKED ISSUE #9: Enhance swap command to support an empty destination
image

### PR DESCRIPTION
**Pull request by stdavis**
Closes #9

### CODE REVIEW

**Review by jacobdadams — APPROVED** [review]
Makes sense to me 👍

File: src/swapper/swapper.py [hunk 1]
```diff
@@ -97,13 +96,21 @@ def copy_and_replace( #: pylint: disable=dangerous-default-value too-many-statem
     def check_table_existence(workspace, table_name):
         with arcpy.EnvManager(workspace=workspace):
             if not arcpy.Exists(table_name):
-                raise Exception(f'{table_name} does not exist in {workspace}')
+                raise FileNotFoundError(f'{table_name} does not exist in {workspace}')
 
     check_table_existence(source_workspace, source_feature_class_name)
```
Comment (steveoh) [inline src/swapper/swapper.py:101]: **question**: are we wanting exceptions on empty source to fail the script or should this also be try excepted?

**Review by steveoh — APPROVED** [review]

File: src/swapper/swapper.py (hunk 1 above)
Comment (stdavis) [reply src/swapper/swapper.py:101]: I think that we do want empty source to exit swapper with an exception. This shouldn't short circuit forklift as [the pallet does a try/catch](https://github.com/agrc/warehouse/blob/b7e1dee4a19132c4b97713123b1e3c9dc1bb71a5/sgid/SwapperPallet.py#L53).

**Comment by stdavis** [comment]
I've updated swapper on the forklift server.

**Pull request merged**